app/controllers/patients_controller.rb

```diff
@@ -236,8 +236,10 @@ def update
 
     # Update patient history with detailed edit diff
     patient_before = patient.dup
-    Patient.detailed_history_edit(patient_before, patient, allowed_params&.keys, current_user.email) if patient.update(content)
 
+    render(json: patient.errors, status: 422) && return unless patient.update(content)
```

Comment:
Should be `... and return` rather than `... && return`

Reply:
Addressed in https://github.com/SaraAlert/SaraAlert/pull/582/commits/111784ddc5397e42ab85ecf9025f1d41b90a8ce2.